### PR TITLE
give admins and artists an entry in the artist table

### PR DESCRIPTION
**Pull request by RubenHuizenga**
This pull request applies to the following story: No story, bug fix

## Description
From the admin user screen artists and admins get a entry in the artist table

**Changes proposed in this pull request (technical):**
* From the admin user screen artists and admins get a entry in the artist tables
*

## Checklist:
- [ ] :nerd_face: Documentation for this PR is covered.
- [ ] :clipboard: All tests are correct / working

### CODE REVIEW

**Review by BennedYT — APPROVED** [review]

**Review by Rblindeboom — APPROVED** [review]

**Review by Rblindeboom — APPROVED** [review]
mooi man

**Review by BennedYT — DISMISSED** [review]

File: View/Screens/RoleAssignmentScreen.xaml.cs [hunk 1]
```diff
@@ -45,8 +45,12 @@ private void UserRole_SelectionChanged(object sender, SelectionChangedEventArgs
         {
             User user = (User)((ContentPresenter)((ComboBox)sender).TemplatedParent).Content;
             int roleID = ((ComboBox)sender).SelectedIndex + 1;
+            var artistController = ArtistController.Create(DatabaseContext.Instance);
             if(user.RoleID != roleID)
             {
+                if (user.RoleID == 1) artistController.CreateItem(new Artist {ArtistName = user.Username, UserID = user.ID});
```
Comment (PaulRill00) [inline View/Screens/RoleAssignmentScreen.xaml.cs:51]: moet dit niet `user.RoleID != 1` zijn?

Comment (RubenHuizenga) [reply View/Screens/RoleAssignmentScreen.xaml.cs:51]: nope, dit als je user bent en naar artist of admin gaat
dan krijg je een entry in de artist tabel

Comment (PaulRill00) [reply View/Screens/RoleAssignmentScreen.xaml.cs:51]: Maar dan vervolgens, als je roleid 1 is, revoke artist?

Comment (RubenHuizenga) [reply View/Screens/RoleAssignmentScreen.xaml.cs:51]: user.roleID is wat je bent, roleID is wat je wordt

Comment (PaulRill00) [reply View/Screens/RoleAssignmentScreen.xaml.cs:51]: check, gotcha. Noice

**Review by PaulRill00 — APPROVED** [review]

File: View/Screens/RoleAssignmentScreen.xaml.cs (hunk 1 above)
Comment (RubenHuizenga) [reply View/Screens/RoleAssignmentScreen.xaml.cs:51]: Facebook_logo_thumbs_up_like_transparent

**Review by Rblindeboom — APPROVED** [review]

**Review by BennedYT — APPROVED** [review]

**Pull request merged**